master/usage/configuration/mtu.md

```diff
@@ -82,3 +82,12 @@ calicoctl config set --raw=felix IpInIpMtu 1480
 
 > **Note:** Setting the `IpInIpMtu` config option will result in an immediate
 update the tunnel interface MTU on all of the active nodes in your cluster.
+
+## Kubernetes
```

Comment:
I think this isn't quite the right title - it's more "Configuring MTU in Kubernetes self-hosted manifests"

An alternative to this is to add configuration options to our manifests (set at whatever the default value is) so that people have a knob to turn for this.  I think that's probably a better option.  WDYT?